src/index.js

```diff
@@ -18,7 +18,21 @@ function renderSidebar() {
     <ul class="email-list">${messageLIs}</ul>
   `;
   const container = document.querySelector('.email-list');
-  if (container != null) container.innerHTML += sidebarContents;
+  if (container != null) container.innerHTML = sidebarContents;
 }
 
-renderSidebar();
+function renderMailbox(eventTarget) {
+  const mailboxName = eventTarget.textContent;
+  const activeMB = document.querySelector('.active');
+  activeMB.classList.remove('active');
```

Comment:
This is excellent. I always want to find a way to remove all the other items `active` class. But actually I only need to remove the current `active`. Thank you so much